scripts/update_docs_website.sh

```diff
@@ -0,0 +1,15 @@
+git clone git@github.com:Moya/moya.github.io.git _moya.github.io
```

Comment:
Didn't know if you wanted the SSH url or the HTML one, can change it in case

Reply:
ssh is great, thanks!